types/searchRange.js

```diff
@@ -1,12 +1,13 @@
 const Serializable = require('./serializable')
 
 const keyWhitelist = ['CREATED', 'MODIFIED']
+const sortOrderWhitelist = ['ASCENDING', 'DESCENDING']
 
 /**
  * Create a search query against Tozny Platform.
  */
 class SearchRange extends Serializable {
-  constructor(start, end, key = 'CREATED') {
+  constructor(start, end, sort_order = 'ASCENDING', key = 'CREATED') {
```

Comment:
The new parameter should be placed at the end so that existing implementations don't get unexpected behavior